docs/glyph-inspector.html

```diff
@@ -161,7 +161,7 @@ <h1>Free Software</h1>
         html += '<dt>leftSideBearing</dt><dd>'+glyph.leftSideBearing+'</dd>';
     }
     html += '</dl>';
-    if (glyph.numberOfContours > 0) {
+    if (glyph.numberOfContours > 0 && window.font.variation) {
```

Comment:
But couldn't there be glyphs with multiple contours but no variation axes? So I think the check for `window.font.variation` should happen inside, and depending on its availability, either the regular glyph or the transformed variant should be used.

Reply:
I’m not very familiar with this professional knowledge. I only encountered this issue during use, so there might be some oversights in the fix. Thank you for pointing it out.